tests/integration/arbitrary-pine-requests.spec.ts

```diff
@@ -0,0 +1,114 @@
+import * as m from 'mochainon';
+import { describeExpandAssertions } from '../util_ts';
+import {
+	balena,
+	BalenaSdk,
+	givenAnApplication,
+	givenLoggedInUser,
+} from './setup';
+const { expect } = m.chai;
+
+describe('arbitrary pine requests on organizations model', function() {
```

Comment:
I really don't like the description, can't we find several representative cases and test those instead? Or we can rename it to "the organization model object shape" or something along those lines.

Reply:
@sradevski We actually already have `should support arbitrary pinejs options` in various model tests and that's why I followed a similar naming scheme here.
For first, let me drop the `on organizations model` suffix, since that's too specific.
Secondly, `describeExpandAssertions` is actually creating more sensible test cases and the result that it prints looks something like:
```
  arbitrary pine requests
    expanding from application
      to organization
        ✓ should succeed (535ms)
        ✓ should include the expanded property
      to user__is_member_of__application
        ✓ should succeed (536ms)
        ✓ should include the expanded property
      to team_application_access
        ✓ should succeed (685ms)
        ✓ should include the expanded property
    expanding from user__is_member_of__application
      to user
        ✓ should succeed (523ms)
        - should include the expanded property
      to is_member_of__application
        ✓ should succeed (518ms)
        - should include the expanded property
      to application_membership_role
        ✓ should succeed (527ms)
        - should include the expanded property
```

The whole point of this PR was to make sure that our typings are still valid after the (hopefully last) upcoming model changes.
If you feel that it's better to not merge it, I can definitely just keep rebasing it for testing purposes and just close it once orgs get merged.

> can't we find several representative cases and test those instead?

Can I get an example?

Reply:
@thgreasi ah I see what's your goal here. As long as we cleanup afterwards, I am fine with it as it is.